user/main.py

```diff
@@ -16,7 +18,53 @@ def get_db():
     finally:
         db.close()
 
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+    
+# To insert new user into the table
+
+@app.post("/create", status_code=status.HTTP_201_CREATED)
+def create_user(request: schemas.User, db: Session=Depends(get_db)):
+    hashed_password = pwd_context.hash(request.password)
+    new_user = models.User(name=request.name, email=request.email, password=hashed_password)
+    if new_user:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST,detail="User already exists")
+    else:
+        db.add(new_user)
+    db.commit()
+    db.refresh(new_user) 
+    return "New user is added successfully"
+    
+# To get all the user details
+@app.get("/read", status_code=status.HTTP_200_OK)
+def read_users(db: Session=Depends(get_db)):
+    user_list = db.query(models.User).all()
+    return user_list
```

Comment:
We can skip the variable here and directly return the `db.query(models.User).all()`.